wolfssl/ssl.h

```diff
@@ -287,6 +289,17 @@ struct WOLFSSL_GENERAL_NAME {
     } d; /* dereference */
 };
 
+struct WOLFSSL_DIST_POINT_NAME {
+    int type;
+    union {
+        WOLF_STACK_OF(WOLFSSL_GENERAL_NAME)* fullname;
```

Comment:
Prefer camel case, so `fullName`, unless I'm wrong on the preferred style.

Reply:
in the Python port the code is expecting the name fullname, commit https://github.com/wolfSSL/wolfssl/pull/4351/commits/fdad39b879717000f8f45111e720be90dff96035 adds in a comment